contrib/clblast/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Start with Ubuntu 23.04 as base image
+FROM rust as build
```

Comment:
https://hub.docker.com/layers/library/rust/latest/images/sha256-bd94b2d02b52595a8759a0cad5ab5c90c28e35504eefc923365aa091e5ef20c9?context=explore

You might wanna update base image or comments here?

Reply:
Changed to rust:latest